src/components/Avatar.js

```diff
@@ -1,10 +1,10 @@
+const Avatar = ({ image, name = "Twitter Mentors" }) => {
```

Comment:
Why the default parameter value?
Is this for if the API does not return a user profile name

Reply:
This `Avatar` component is shared between the `Sidebar` and `Tweets` section. 
On the Sidebar top, It shows the app name and on the right Tweets section, it gets the user name.

Reply:
Understood, great logical thinking!